gradle-palantir-java-format/build.gradle

```diff
@@ -88,3 +88,7 @@ def writeImplClasspath = tasks.register("writeImplClasspath") {
 tasks.named("test").configure {
     dependsOn(writeImplClasspath)
 }
+
+tasks.withType(ProcessResources.class).configureEach {
+    from(project(':palantir-java-format').tasks.named('nativeCompile')) {}
+}
```

Comment:
We'll want to download them separately using a configuration right? Rather than just shoving them in the jar and making it enormous

Reply:
Yes, I needed to include the native image in the resources for some tests. I will push a change which will only affect the test resources.